packages/wonder-blocks-clickable/src/components/__docs__/clickable.stories.js

```diff
@@ -0,0 +1,296 @@
+// @flow
+import * as React from "react";
+
+import {StyleSheet} from "aphrodite";
+import {MemoryRouter, Route, Switch} from "react-router-dom";
+
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {Body, LabelLarge} from "@khanacademy/wonder-blocks-typography";
+
+import type {StoryComponentType} from "@storybook/react";
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+import argTypes from "./clickable.argtypes.js";
+
+export default {
+    title: "Clickable / Clickable",
+    component: Clickable,
+    argTypes: argTypes,
+    args: {
+        testId: "",
+        disabled: false,
+        light: false,
+        hideDefaultFocusRing: false,
+    },
+    decorators: [
+        (Story: StoryComponentType): React.Element<typeof View> => (
+            <View style={styles.centerText}>
+                <Story />
+            </View>
+        ),
+    ],
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
+            description: {
+                component: null,
+            },
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+};
+
+export const Default: StoryComponentType = (args) => (
```

Comment:
Oh neat, how did you get the actions tab to work here without importing storybook actions?

Reply:
I added this key in the `onClick` argTypes object: https://github.com/Khan/wonder-blocks/pull/1414/files#diff-ad027f4945984ad8939691c76e61b40fb881603b336ec432b8dfddbd4201f93bR105